apps/pom.xml

```diff
@@ -650,6 +654,17 @@ checkJavaFormat
 			</plugins>
 		</pluginManagement>
 		<plugins>
+			<!-- Use the versions-maven-plugin plugin to help ensure that checking
+							for new versions to update dependencies to don't include alphas.
+							 In the apps directory, run: "mvn versions:display-dependency-updates"-->
```

Comment:
Wording suggestion:

This plugin configuration allows us to scan all of the pom files to find any dependencies that have newer versions available that are not alpha, beta, or other commonly used pre-release version tags. To invoke this plugin to find the out of data dependencies, run: "mvn versions:display-dependency-updates" from the apps folder.

(also, please use a fixed indentation level)